openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -107,7 +107,7 @@ def _add_toggle_instance_details(self, toggle, toggle_instance):
         """
         toggle['class'] = toggle_instance.__class__.__name__
         toggle['module'] = toggle_instance.module_name
-        if is_code_owner_mappings_configured():
+        if toggle_instance.module_name:
```

Comment:
Close.  We want `toggle['module'] = toggle_instance.module_name` to be after this check, right?

Reply:
I don't think so, no, because that would change the existing behaviour.

Reply:
We're defining the behavior. :)  Again, this was written hackathon style.  I don't see why we would skip `code_owner` when we don't have it, but not do the same for `module`.

Reply:
We can always follow-up if you don't want this in this PR for some reason. 
1. Please do this or don't. :)
2. Squash as you wish.
3. Let me know it is ready for merge.
I will retest in the morning and merge hopefully.  Thanks for taking care of this.

Reply:
There are indeed many things in this module which could be improved, but I don't want us to move off-track in this PR which has already grown beyond its original intended scope, and which I need to merge to start working on other stuff...